arm/ui/database/database.py

```diff
@@ -46,7 +46,8 @@ def view_database():
     # Check for database file
     if os.path.isfile(cfg.arm_config['DBFILE']):
         jobs = models.Job.query.order_by(db.desc(models.Job.job_id)).paginate(page=page,
-                                                                              max_per_page=int(armui_cfg.database_limit),
+                                                                              max_per_page=int(
```

Comment:
This change has no effect?

Reply:
Line length for the Quality checks. Without this change, it causes the checks to fail.

Max line length is 120, but without this change, the total line length is 121.

Reply:
Ah, fair enough